partcad/src/partcad/adhoc/convert.py

```
@@ -0,0 +1,86 @@
+    config_path = temp_dir / "partcad.yaml"
+    config_content = f"""
+parts:
+  input_part:
+    type: {input_type}
+    path: {temp_input_path}
+    """
+    with open(config_path, "w") as config_file:
+        config_file.write(config_content)
+
+
+def export_part(part, output_filename, output_type, ctx):
```

Ruff (0.8.2)

Missing return type annotation for public function `export_part`

Add return type annotation: `None`

(ANN201)